server/protocols/webadmin/webadmin-data/src/test/java/org/apache/james/webadmin/routes/AddressMappingRoutesTest.java

```diff
@@ -70,6 +72,7 @@ void setUp() throws Exception {
 
         RestAssured.requestSpecification = WebAdminUtils.buildRequestSpecification(webAdminServer)
             .setBasePath(AddressMappingRoutes.BASE_PATH)
+            .setUrlEncodingEnabled(false) // no further automatically encoding by Rest Assured client
```

Comment:
Put the ticket number in the comment ?

`// JAMES-3936 no further automatically encoding by Rest Assured client`